### DISCUSSION TITLE
Nested file's alignment is affecting parent leveled file's alignment

### DISCUSSION THREAD

**Discussion by curiousDev288** [microsoft/vscode-discussions#1102] [Extension Development QnA] [answered]
I have couple of nested files & folders in my extension. On expanding to the depth/level 4 folder, nested files get aligned to right along with parent leveled files. 
On collapsing again they gets into expected view.

PFB snapshot on expanding folder 'folder' : 
before : 
<img width="148" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/100760141/32ae3747-07a9-4997-bb2e-b33fd8d10295">

after : 
<img width="142" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/100760141/0ea21636-cf40-4b3e-9ced-e6044cbd3f84">

Expected : 
After collapse/expand files must not have impact on parent level files.

Kindly suggest solution for this issue

**Comment by gjsjohnmurray** [comment, marked as answer]
Please see https://github.com/microsoft/vscode/issues/148974#issuecomment-1315304859

Please also not that this discussion forum is currently focused on developers of extensions, and the repository linked above is a better place for issues like yours.

**Reply by curiousDev288** [reply]
I tried solution mentioned in [https://github.com/microsoft/vscode/issues/148974#issuecomment-1315304859](https://github.com/microsoft/vscode/issues/148974#issuecomment-1315304859), but issue still persists.
For the nested level 4 and above issue is there.
